CONTRIBUTING.md

```diff
@@ -22,3 +25,8 @@ Currently the following reference material is pulled from other repos:
 - [Python SDK](https://www.balena.io/docs/reference/sdk/python-sdk/) sourced from https://github.com/balena-io/balena-sdk-python/blob/master/DOCUMENTATION.md
 
 Version numbers and commit messages are automatically added to the `CHANGELOG.md` file by the CI build flow, after a pull request is merged. It should not be manually edited.
+
+asdasd
+blah 
```

Comment:
blah blah? :wink:

Reply:
@stathismor No worries, I was testing the methods one can add `Change-type` on their commits to inform versionbot of the change-type for semantic versioning. Re:  `versionbot-pr-draft` label, it makes sure the PR can't be auto-merged by the balena-bulldozer until the label is present on a PR. Resolving. 

cc: [Discussion Thread](https://www.flowdock.com/app/rulemotion/r-beginners/threads/LDHkRR-FUcDF04VCtRkNWUvc-Zc)